doc/devdocs/modules/FileExplorer/monaco/readme.md

```diff
@@ -0,0 +1,60 @@
+# Developer Preview (Monaco)
+
+Developer preview is based on [Microsoft's Monaco Editor](https://microsoft.github.io/monaco-editor/) which is maintained by the Visual Studio Code team.
+
+## Update Monaco Editor
+
+1. Download Monaco editor with [npm](https://www.npmjs.com/): `npm i monaco-editor`.
+2. Delete everything except the `min` folder (the minimised code) from the downloaded files.
+3. Copy the `min` folder inside the [`monacoSRC`](/src/modules/previewpane/MonacoPreviewHandler/monacoSRC) folder.
```

Comment:
What is the copy target?

Reply:
Oh you're right. This should be "into" and not "inside"